challenger/challenger-processor/challenger_task_processor.go

```diff
@@ -0,0 +1,65 @@
+package challengerprocessor
+
+import (
+	"fmt"
+
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	sdktypes "github.com/Layr-Labs/eigensdk-go/types"
+)
+
+type IndexingChallengerProcessor[Input any, Output any] struct {
+	logger               logging.Logger
+	responseValidationFn ResponseValidationFunction[Input, Output]
+	challengerRaiser     ChallengerRaiser[Input, Output]
+	tasks                map[uint32]sdktypes.GenericInputTask[Input]
+}
+
+type ResponseValidationFunction[Input any, Output any] func(taskIndex uint32, input Input, output Output) (bool, error)
+
+type ChallengerRaiser[Input any, Output any] interface {
```

Comment:
Let's rename this to `ChallengeRaiser`.
```suggestion
type ChallengeRaiser[Input any, Output any] interface {
```

Reply:
Done in [1a64631](https://github.com/Layr-Labs/eigensdk-go/pull/672/commits/1a6463146b3f94fbec245277e980d8469b12b15f)